o4/o4.py

```diff
@@ -1187,13 +1194,46 @@ def move_except(from_dir, to_dir, but_not):
     check_call(cmd)
     if not discard:
         savedir = source.replace('cleaning', time.strftime('cleaned'))  # @%Y-%m-%d,%H:%M'))
+        if os.path.exists(savedir):
```

Comment:
This can't happen now, right?

Reply:
I think it can; at least, I think I was seeing it. I was very confused.